packages/web-components/fast-components/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -0,0 +1,141 @@
+import { css } from "@microsoft/fast-element";
+import { display, focusVisible } from "../../styles";
+import {
+    neutralDividerRestBehavior,
+    neutralFocusBehavior,
+    neutralForegroundActiveBehavior,
+    neutralForegroundFocusBehavior,
+    neutralForegroundRestBehavior,
+} from "../../styles/recipes";
+import { heightNumber } from "../../styles/size";
+import { forcedColorsStylesheetBehavior } from "../../styles/match-media-stylesheet-behavior";
+import { SystemColors } from "../../styles/system-colors";
+
+export const AccordionItemStyles = css`
+    ${display("flex")} :host {
+        box-sizing: border-box;
+        font-family: var(--body-font);
+        flex-direction: column;
+        ${
+            /* Font size, weight, and line height are temporary - 
+            replace when adaptive typography is figured out */ ""
+        } font-size: 12px;
+        font-weight: 400;
+        line-height: 18px;
+        border-bottom: calc(var(--outline-width) * 1px) solid var(--neutral-divider-rest);
+    }
+    
+    .region {
+        display: none;
+        padding: calc((6 + (var(--design-unit) * 2 * var(--density))) * 1px);
+    }
+
+    .heading {
+        display: grid;
+        position: relative;
+        grid-template-columns: auto 1fr auto calc(${heightNumber} * 1px)
+    }
+
+    .button {
+        appearance: none;
+        border: none;
+        background: none;
+        grid-column: 2;
+        grid-row: 1;
+        outline: none;
+        padding: 0 calc((6 + (var(--design-unit) * 2 * var(--density))) * 1px);
+        text-align: left;
+        height: calc(${heightNumber} * 1px);
+        color: var(--neutral-foreground-rest);
+        cursor: pointer;
+    }
+
+    .button:hover {
+        color: var(--neutral-foreground-hover);
+    }
+
+    .button:active {
+        color: var(--neutral-foreground-active);
+    }
+
+    .button::before {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        z-index: 1;
+        cursor: pointer;
+    }
+
+    .button:${focusVisible}::before {
+        outline: none;
+        border: calc(var(--outline-width) * 1px) solid var(--neutral-focus);
+        box-shadow: 0 0 0 calc((var(--focus-outline-width) - var(--outline-width)) * 1px)
+            var(--neutral-focus);
+    }
+
+    :host(.expanded) .region {
+        display: flex;
+    }
+
+    .heading {
+        z-index: 2;
+    }
+
+    .glyph {
```

Comment:
I think we might want to change all references to glyph to be "icon": https://github.com/WICG/open-ui/issues/27#issuecomment-602735148

I believe glyph is typically referring to a specific icon.